ibis/tests/expr/test_table.py

```diff
@@ -23,37 +23,26 @@
 from ibis.expr.types import Column, Table
 from ibis.tests.util import assert_equal, assert_pickle_roundtrip
 
-
-@pytest.fixture
-def set_ops_schema_top():
-    return [("key", "string"), ("value", "double")]
-
-
-@pytest.fixture
-def set_ops_schema_bottom():
-    return [("key", "string"), ("key2", "string"), ("value", "double")]
+set_ops_schema_top = [("key", "string"), ("value", "double")]
+set_ops_schema_bottom = [("key", "string"), ("key2", "string"), ("value", "double")]
+setops_relation_error_message = "Table schemas must be equal for set operations"
```

Comment:
These changes seem like churn for no obvious reason. Can we just leave things as they were?

Reply:
VSCode doesn't give type hints for pytest fixtures, but it does for literals. That's how I found this and why I changed it. I can leave it out though if you still want

Reply:
Ok, it seems fine then.